stewards.json

```diff
@@ -26,5 +26,9 @@
     {
         "ca": "lavender_five/server_ca.crt",
         "endpoint": "https://rpc.sommelier.circlewithadot.net:5734"
-    }
-]
\ No newline at end of file
+    },
+    {
+    "ca": "stakecito/server_ca.crt",
+    "endpoint": "https://steward.stakesandstone.com:5734"
```

Comment:
Couple issues here:

1. This is breaking the indentation of the file, will need to be updated.
2. I tested a connection to your endpoint and the response began with this error:

```
$ openssl s_client -connect steward.stakesandstone.com:5734 -CAfile stakecito.crt
CONNECTED(00000005)
depth=1 C = AU, ST = Some-State, O = Internet Widgits Pty Ltd, CN = steward.stakesandstone.com
verify return:1
depth=0 C = AU, ST = Some-State, O = Internet Widgits Pty Ltd, CN = steward.stakesandstone.com
verify return:1
4333143424:error:1417B07B:SSL routines:tls_process_cert_verify:bad signature:ssl/statem/statem_lib.c:505:
```

It seems like something is malformed in your certificates, perhaps your server certificate was not correctly signed by the CA for some reason? I haven't seen that happen using our script. I would recommend generating a fresh set and re-opening a new (single) PR with the certs and the above indentation issue fixed.